content/blog/rust-gamedev-meetup-34/index.md

```diff
@@ -0,0 +1,83 @@
++++
+title = "Rust Gamedev Meetup 34"
+date = 2024-02-10
+transparent = true
+aliases = ["posts/gamedev-meetup-34"]
++++
+
+<!-- markdownlint-disable single-title heading-increment -->
+<!-- markdownlint-disable no-blanks-blockquote no-emphasis-as-header -->
+<!-- markdownlint-configure-file {"line-length": {"heading_line_length": 120}} -->
+
+![Rust Gamedev Meetup](gamedev-meetup.png)
+
+The 34th Rust Gamedev Meetup took place on TODO. You can watch the
+recording of the meetup [here on YouTube][meetup-video]. The meetup took
+place on the Rust Gamedev [YouTube][youtube-stream] and [Twitch][twitch-stream].
+The meetups take place on the second Saturday of every month via the [Rust
+Gamedev Discord server][rust-gamedev-discord] and are also [streamed on
+Twitch][rust-gamedev-twitch].
+
+[rust-gamedev-discord]: https://discord.gg/yNtPTb2
+[rust-gamedev-twitch]: https://twitch.tv/rustgamedev
+[meetup-video]: https://www.youtube.com/watch?v=I_7AgjiE9RA
+[youtube-stream]: https://www.youtube.com/@RustGameDevelopment
+[twitch-stream]: https://www.twitch.tv/RustGameDev
+
+## Talks
+
+### [Cushy][cushy-video] | [@khonsulabs]
+
+[![Cushy](cushy.png)][cushy-video]
+
+> TODO
+
+**Links:**
+
+- [@khonsulabs]
+- [Cushy GitHub repository]
+- [Presentation Source]
+- [Ecton on Mastodon]
+- [Khonsu Labs Discord]
+
+[@khonsulabs]: https://github.com/khonsulabs
+[cushy-video]: https://www.youtube.com/watch?v=aZxtzbR6Xn8
+[Cushy GitHub repository]: https://github.com/khonsulabs/cushy
+[Presentation Source]: https://github.com/khonsulabs/cushy-show/blob/main/examples/introducing_cushy/main.rs
+[Ecton on Mastodon]: https://fosstodon.org/@ecton
+[Khonsu Labs Discord]: https://discord.khonsulabs.com/
+
+### [Astratomic]][astratomic-video] | [@spicylobstergames]
+
+[![Astratomic](astratomic.png)][astratomic-video]
+> TODO
+
+**Links:**
+
+- [@spicylobstergames]
+- [Astratomic GitHub repository]
+- [Spicy Lobster Discord]
+
+[@spicylobstergames]: https://github.com/spicylobstergames
+[astratomic-video]: https://www.youtube.com/watch?v=tTiV59W34JI
+[Astratomic GitHub repository]: https://github.com/spicylobstergames/astratomic
+[Spicy Lobster Discord]: https://discord.com/invite/JFhxYBvxR8
+
+### [Graphite][graphite-video] | [@GraphiteEditor]
+
+[![Graphite](graphite.png)][graphite-video]
+> TODO
```

Comment:
```suggestion
[![Graphite](graphite.png)][graphite-video]

> Graphite, a Rust-based 2D graphics editor, presents the past month of development progress
```

Reply:
@Keavon does this look alright to you?

Reply:
I'm not quite sure what I am looking at. IIRC, I never submitted a PR for this old stalled newsletter. It looks like this suggested change just adds one sentence of text? Would you like me to retroactively submit the content for this old newsletter now that it's being resuscitated?

Reply:
This is not the newsletter, but the post about the meetup from a month ago ;) 
The sentence you're reading is just a small description of what you presented then.

Reply:
Oh I see, there are blog posts being added about each newsletter now?

Reply:
As far as I can tell, this has been the case since September: https://gamedev.rs/blog/

I think the value comes from having a link to the recording of the individual talks.

Reply:
If this is about last month's meetup, let's have it say this instead because it encompasses more like half a year, or arguably a full year, of work:

> Graphite, a Rust-based 2D creative tool for procedural graphics editing, presents a recap of development accomplishments from 2023